src/redux/reducers.js

```diff
@@ -4,7 +4,8 @@ import jwt_decode from 'jwt-decode';
 const user = (state = [], action) => {
   switch(action.type) {
     case 'SET_TOKEN':
-      const decoded = jwt_decode(action.token)
+      console.log(action.token);
```

Comment:
this is also my bad for leaving console logs in.. should we take both out?

Reply:
done